orm/src/test/java/org/hibernate/search/test/bridge/tika/TikaBridgeSupportTest.java

```diff
@@ -38,7 +38,7 @@
 /**
  * @author Hardy Ferentschik
  */
-public class TikaBridgeBlobSupportTest extends SearchTestBase {
+public class TikaBridgeSupportTest extends SearchTestBase {
```

Comment:
Did you see there already are *two* classes named `TikaBridgeTest`? Maybe introducing a third class with a name this generic is not a good idea...
What would you think of "TikaBridgeInputFormatTest"?